### ISSUE TITLE
[Bug]: RBWM now 404's after they updated website

### ISSUE THREAD

**Issue by naigle** [mampfes/hacs_waste_collection_schedule#2385]
### I Have A Problem With:

A specific source

### What's Your Problem

Source specific issue... they (RBWM - Windsor and Maidenhead - UK) have just updated their website, so suspect they have changed other details!

Just getting 404 error at teh moment

### Source (if relevant)

my.rbwm.gov.uk

### Logs

```Shell
This error originated from a custom integration.

Logger: waste_collection_schedule.source_shell
Source: custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py:147
integration: Waste Collection Schedule (documentation)
First occurred: 21:39:58 (1 occurrences)
Last logged: 21:39:58

fetch failed for source Windsor and Maidenhead: Traceback (most recent call last): File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py", line 145, in fetch entries = self._source.fetch() ^^^^^^^^^^^^^^^^^^^^ File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source/rbwm_gov_uk.py", line 41, in fetch r.raise_for_status() File "/usr/local/lib/python3.12/site-packages/requests/models.py", line 1024, in raise_for_status raise HTTPError(http_error_msg, response=self) requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://forms.rbwm.gov.uk/404
```

### Relevant Configuration

```YAML
NA
```

### Checklist Source Error

- [ ] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [X] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [ ] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by 5ila5** [comment]
fixed in #2390 

This will be part of the next release or is available now as master (git) version (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)

You will need to modify your configuration (add postcode argument)

**Comment by naigle** [comment]
Confirmed all working with master
